database/awsImageSeeding.js

```diff
@@ -0,0 +1,67 @@
+const AWS = require('aws-sdk');
+const faker = require('faker');
+const fetch = require('node-fetch');
+const fs = require('fs');
+const path = require('path');
+
+const { AWSAccessKeyId, AWSSecretKey } = require('./aws.config.js');
+AWS.config.update({
+  accessKeyId: AWSAccessKeyId,
+  secretAccessKey: AWSSecretKey,
+  region: 'us-west-2'
+});
+
+const Bucket = 'fec-overview';
+const s3Bucket = new AWS.S3({
+  params: { Bucket }
+});
+const s3Url = 'https://fec-overview.s3-us-west-2.amazonaws.com/'
+
+const imageToS3 = (image, i) => {
+  const pathName = 'ownerPics/' + i.toString() + '.jpg';
+  const data = {
+    Key: pathName,
+    Body: image,
+    ContentType: 'image/jpeg',
+    ACL: 'public-read'
+  };
+  return new Promise((resolve, reject) => {
+    s3Bucket.putObject(data, err => {
+      err ? reject(err) : resolve(s3Url + pathName)
+    })
+  })
+};
+
+const seedPhotos = async () => {
```

Comment:
I'm still confused on how you're seeding this specifically to the campId... but this looks great.

Reply:
That portion gets taken care of on the seeder.js code on that line i showed you yesterday... I basically have a for loop that lopops from 0-99 and campId's 1 thru 99 have an image on the s3 server that is just 1.jpg, 2.jpg, ... 99.jpg, and the URL gets saved to the DB. if you want you can run the code after you drop the current copies of my DB that you have and then look at what I'm doing.